src/main/java/com/druppel/api/dal/MeasurementRepo.java

```diff
@@ -23,14 +23,13 @@
 
     /**
      * Returns list of average of measured values grouped by day and type on a specific time interval
-        * using interface-based Data JPA Projections
-     @param  date1 Date for date where the values were measured
-     @param  date2 Date for date where the values were measured
-     @return  List of date,type, average value
+     * using interface-based Data JPA Projections
+     *
+     * @param days the amount of days you want
+     * @param espId the esp you want to query on
+     * @return List of date,type, average value
      */
-
-    @Query("Select date As date, type As type, avg (value) AS value From Measurement Where date >= ?1 and date<=?2 and espId =?3 group by type,date" )
-    public List<MeasurementSummary> getAverageSummary(Date date1, Date date2,int espId);
-
+    @Query("SELECT date_created, type, value FROM v_measurement WHERE date_created > current_date - interval ?1 day AND esp_id = ?2" )
```

Comment:
Omdat we Spring gebruiken, dient in de query alleen de entity object gebruikt worden.
De view tabel dient eerst naar Jpa Object omgezet(mappen) , die object kan vervolgens in de query gebruikt worden.
Zorg ook dat Id in de v_measurement staat , het is een eis van Spring. bijvoorbeeld door de currrent number row te gebruiken als Id ( SELECT rownum AS ID)
De methode retourneert list<MeasurementSummary > is prima